edbdeploy/data/ansible/kubernetes_roles/remove_cloudnativepostgres_sandbox/README.md

```diff
@@ -0,0 +1,68 @@
+# remove_cloudnativepostgres_sandbox
```

Comment:
Duplicate role in edb-ansible:
https://github.com/EnterpriseDB/edb-ansible/tree/master/roles/remove_cloudnativepostgres_sandbox